ci_framework/roles/artifacts/tasks/cluster_info.yml

```diff
@@ -89,3 +89,52 @@
+      ansible.builtin.shell: |
+        echo "cifmw_install_yamls_vars:" > {{ cifmw_artifacts_basedir }}/artifacts/operator_images.yaml
+
+    - name: Get operator images from "{{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }}" namespace
+      changed_when: true
+      ignore_errors: true
+      environment:
+        KUBECONFIG: "{{ cifmw_install_yamls_environment['KUBECONFIG'] }}"
+        PATH: "{{ cifmw_path }}"
+      ansible.builtin.shell: |
+        # openstack-operator-index image
+        pod_name=$(oc get pods -l olm.catalogSource=openstack-operator-index --no-headers=true -n {{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }} | awk '{print $1}');
+        image=$(oc get pod ${pod_name} -n {{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }} -o jsonpath='{.status.containerStatuses[0].imageID}');
+        echo "  OPENSTACK_IMG: ${image}" >> {{ cifmw_artifacts_basedir }}/artifacts/operator_images.yaml;
+        # openstack-operator image
+        pod_name=$(oc get pods -l openstack.org/operator-name=openstack --no-headers=true -n {{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }} | awk '{print $1}' );
+        image=$(oc get pod ${pod_name} -n {{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }} -o jsonpath='{.status.containerStatuses[1].imageID}');
+        echo "  OPENSTACK_OP_IMG: ${image}" >> {{ cifmw_artifacts_basedir }}/artifacts/operator_images.yaml;
+        # rabbitmq-operator image
+        pod_name=$(oc get pods -l app.kubernetes.io/component=rabbitmq-operator --no-headers=true -n {{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }} | awk '{print $1}');
+        image=$(oc get pod ${pod_name} -n {{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }} -o jsonpath='{.status.containerStatuses[0].imageID}');
+        echo "  RABBITMQ_OP_IMG: ${image}" >> {{ cifmw_artifacts_basedir }}/artifacts/operator_images.yaml;
+
+        # Rest of the operator images
+        for pod in `oc get pod -l openstack.org/operator-name --no-headers=true -n {{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }} | grep -v 'openstack-operator-controller-manager' | awk '{print $1}'`;  do
+            operator_name=$(oc get pod $pod -o jsonpath='{.metadata.labels.openstack\.org/operator-name}' -n {{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }} --no-headers=true | tr '[:lower:]' '[:upper:]');
+            image_name=$(oc get pod $pod -n {{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }} -o jsonpath='{.status.containerStatuses[1].image}' );
+            echo "  ${operator_name}_OP_IMG: ${image_name}" >> {{ cifmw_artifacts_basedir }}/artifacts/operator_images.yaml;
+         done
+
+    - name: Get openstack service images from "{{ cifmw_install_yamls_defaults['NAMESPACE'] }}" namespace
+      changed_when: true
+      ignore_errors: true
+      environment:
+        KUBECONFIG: "{{ cifmw_install_yamls_environment['KUBECONFIG'] }}"
+        PATH: "{{ cifmw_path }}"
+      ansible.builtin.shell: |
```

Comment:
ditto - using `ci_script` would give better logging and reproducer capabilities.

Reply:
Updated